Mathlib/Algebra/Order/WithTop/Untop0.lean

```diff
@@ -65,6 +65,9 @@ lemma untopD_add [Add α] {a b : WithTop α} {c : α} (ha : a ≠ ⊤) (hb : b 
 lemma untop₀_add [AddZeroClass α] {a b : WithTop α} (ha : a ≠ ⊤) (hb : b ≠ ⊤) :
     (a + b).untop₀ = a.untop₀ + b.untop₀ := untopD_add ha hb
 
+@[simp]
+lemma untop₀_natCast [AddMonoidWithOne α] (n : ℕ) : untop₀ (n : WithTop α) = n := rfl
```

Comment:
That sounds like an exam now, but could you add the `ofNat` version? 😁

Reply:
Unable. I tried, but always ended up with the message
```
failed to synthesize
  OfNat α n
```
I guess I failed the exam. Please advise.